src/EnergyPlus/SetPointManager.cc

```diff
@@ -5341,7 +5358,24 @@ namespace SetPointManager {
 			this->MySetPointCheckFlag = false;
 		}
 
-		this->SetPt = Node( RefNode ).TempSetPoint - ( Node( FanOutNode ).Temp - Node( FanInNode ).Temp );
+		if ( CoolCoilInNode == 0 && CoolCoilOutNode == 0 ) {
+			this->SetPt = Node( RefNode ).TempSetPoint - ( Node( FanOutNode ).Temp - Node( FanInNode ).Temp );
```

Comment:
When you say "differences of example runs are found", what do you mean?  Differences in the CoolingCoilFreezePrevention file?  That would be OK.  Differences in other files would not.

Minor quibble with the code above, the "if ( this->FreezeCheckEnable)" isn't necessary, just let the prior if block continue.